src/libxtp/gwbse/gw.cc

```diff
@@ -235,7 +235,7 @@ Eigen::VectorXd GW::SolveQP(const Eigen::VectorXd& frequencies) const {
     }
   }
 
-  if ((converged == true).all()) {
+  if ((converged == false).any()) {
```

Comment:
Couldn't it be:
`!converged.all()`

Reply:
good question. I am not sure and a bit lazy to try now.

Reply:
you are right and it looks much cleaner.

Reply:
thanks